### PR TITLE
fixing page limit in page.view.tree

### PR DESCRIPTION
**Pull request by hbaptiste**
fixing page limit in page tree.  Defaut is 25.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:

File: src/tb/apps/page/views/page.view.tree.js [hunk 1]
```diff
@@ -63,7 +62,15 @@ define(
                 var config = {
                     dragAndDrop: true,
                     onCreateLi: this.onCreateLi,
-                    id: this.config.popinId || 'bb-page-tree'
+                    id: this.config.popinId || 'bb-page-tree',
+                    onCanMove: function (node) {
+                        if (node.is_fake) {
+                            return false;
+                        }
+                        if (node.hasOwnProperty("has_ellipsis") && node.has_ellipsis === true) {
```
Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.tree.js]: true === node.has_ellipsis is enough no ?

File: src/tb/apps/page/views/page.view.tree.js [hunk 2]
```diff
@@ -245,7 +255,7 @@ define(
             * @param {Object} node
             */
             insertDataInNode: function (data, node) {
-                var key, formattedNode, children = node.children;
+                var key, formattedNode, provNode, ellipsisNode = null, children = node.children;
```
Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.tree.js]: it's prevNode no ?

**Comment by fkroockmann** [comment]
thanks @hbaptiste

**Pull request merged**